doc/liblouis.texi

```diff
@@ -1169,6 +1169,19 @@ This is an opcode prefix which modifies the operation of the opcode
 following it on the same line. nofor specifies that forward translation 
 is not to use the information on this line.
 
+@opcode{nocross, opcode characters ...}
+@code{nocross} is an opcode prefix which modifies the operation of the
+opcode following it. The following opcode can be @opcoderef{always} or
+any other translation opcode listed below with @code{characters} as
```

Comment:
> translation opcode listed below

I don't see any opcodes listed anywhere

Reply:
All the opcodes in the "Translation Opcodes" section.